rasa/nlu/schemas/data_schema.py

```diff
@@ -0,0 +1,86 @@
+from typing import Dict, Text, Any
+
+
+def entity_dict_schema() -> Dict[Text, Any]:
+    """Returns: schema for defining entities in Markdown format."""
+    return {
+        "type": "object",
+        "properties": {
+            "entity": {"type": "string"},
+            "role": {"type": "string"},
+            "group": {"type": "string"},
+            "value": {"type": "string"},
```

Comment:
let's put that in a helper function and re-use them in `rasa_nlu_data_schema` and `entity_dict_schema` instead of duplicating them.